packages/assert/README.md

```diff
@@ -1,10 +1,50 @@
-assert
+@0xproject/assert
 ------
 
 Standard type and schema assertions to be used across all 0x projects and packages
 
-## Install
+## Installation
 
 ```bash
-npm install @0xproject/assert --save
+yarn add @0xproject/assert
+```
+
+## Usage
+
+TODO - Custom documentation page is in progress.
+
+## Contributing
+
+We strongly encourage our community members to help us make improvements and to determine the future direction of the protocol. To report bugs within this package, please create an issue in this repository.
+
+[CONTRIBUTING.md](../../CONTRIBUTING.md)
```

Comment:
`Please read our [contribution guidelines](../../CONTRIBUTING.md) before getting started.`

Reply:
Let's make this change to all of them.